rust-toolchain.toml

```
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.81"
+channel = "1.83"
```

```suggestion
channel = "1.85"
```
Let's go straight to 1.85, since this is the next supported version for zkVM.